pkg/sql/delete.go

```diff
@@ -129,10 +129,6 @@ func (d *deleteNode) Start() error {
 		}
 	}
 
-	if err := d.rh.startPlans(); err != nil {
```

Comment:
This is safe to remove because the visitor stuff checks for subqueries itself now, right?